lib/modules/platform/codecommit/index.md

```diff
@@ -0,0 +1,53 @@
+# AWS CodeCommit
+
+## Authentication
+
+First, you need to obtain an AWS [IAM Access Key id and a Secret access key id](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+
+Let Renovate use AWS CodeCommit access keys by doing one of the following:
+
+- Set a Renovate configuration file - config.js and set:
+  - `endpoint:` the url endpoint e.g `https://git-codecommit.us-east-1.amazonaws.com/`
+  - `username:` AWS IAM access key id
+  - `password:` AWS Secret access key
+- Set environment variables:
+  - `AWS_REGION:` the region e.g `us-east-1`
+  - `AWS_ACCESS_KEY_ID:` your IAM Access key id
```

Comment:
why we need username and access key? we should only require one.

Reply:
it just doesn't work without
need the access key id and secret access key to form a signatureV4 to call http 
the SDK also requires both.

Reply:
take a look
![image](https://user-images.githubusercontent.com/42116482/190883895-1b03deb2-57df-434d-bb2e-897840c8dfcd.png)

Reply:
No, I mean: why set `username` / `password` and  `AWS_ACCESS_KEY_ID` / `AWS_SECRET_ACCESS_KEY`?
I think it should be enouth to have one of those options

Reply:
oh, i had them set differently, but @betterPeleg changed the way it looked he actually says that we need only 1 of them:

`Let Renovate use AWS CodeCommit access keys by doing one of the following:` 

ill add 1 and 2 just to make it clearer.